lego/apps/articles/views.py

```
@@ -52,3 +57,21 @@ def get_object(self) -> Article:
         except (NotAuthenticated, PermissionDenied) as e:
             raise Http404 from e
         return obj
+
+    @action(detail=True, methods=["GET"])
+    def statistics(self, request, pk=None, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `statistics`

(ANN201)